File: Data Structures & Algorithms/minimum-difference-between-highest-and-lowest-of-k-scores/submission-0.py
class Solution:
    def minimumDifference(self, nums: List[int], k: int) -> int:
        nums.sort()
        l = 0
        r = k - 1
        
        minAns = nums[r] - nums[l]

        while r+1 < len(nums):
            r += 1
            l += 1
            val = nums[r] - nums[l]
            minAns = min(val, minAns)
        return minAns

File: Data Structures & Algorithms/concatenation-of-array/submission-0.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int[] newNums = new int[nums.length*2];
        int l = 0;
        int r = nums.length;
        for(int n : nums){
            newNums[l] = n;
            newNums[r] = n;
            l++;
            r++;
        }
        return newNums;
    }
}